CODEOWNERS

```diff
@@ -83,8 +83,8 @@
 /doc/nrf/app_dev/device_guides/nrf91/     @nrfconnect/ncs-modem-doc
 /doc/nrf/app_dev/device_guides/nrf91/nrf91_snippet.rst @nrfconnect/ncs-cia-doc
 /doc/nrf/app_dev/device_guides/pmic/      @nrfconnect/ncs-pmic-doc
-/doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ @FrancescoSer
-/doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_*.rst @nrfconnect/ncs-charon-doc
+/doc/nrf/app_dev/device_guides/nrf54h/ @FrancescoSer
+/doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_suit_*.rst @nrfconnect/ncs-charon-doc
```

Comment:
Is renaming `Working with nRF54H20` -> `Developing with nRF54H20` planned for this PR/task or a following one?

Reply:
The renaming could take place, but it is my understanding that that renaming was connected to the availability of the quick start for the SoC with the precompiled binaries, which unfortunately are not ready yet.

Reply:
Renaming of the section name has nothing to do with the renaming and replacement of the getting started guide.